Application/EdFi.Ods.Admin/Security/IdentityProvider.cs

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using EdFi.Ods.Admin.Contexts;
+using log4net;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Owin.Security;
+
+namespace EdFi.Ods.Admin.Security
+{
+    public class IdentityProvider : IIdentityProvider
+    {
+        private readonly IIdentityContextFactory _identityContextFactory;
+        private readonly ILog _log = LogManager.GetLogger(typeof(IdentityProvider));
+
+        public IdentityProvider(IIdentityContextFactory identityContextFactory)
+        {
+            _identityContextFactory = identityContextFactory;
+        }
+
+        public bool VerifyUserExists(string userEmail)
+        {
+            var identityUserManager = GetIdentityUserManager();
```

Comment:
The pattern in the rest of the class is using this with a "using", should it be done here?